AddShippingZone.tsx

```diff
@@ -1,18 +1,138 @@
-import React from "react";
-import { Button, SafeAreaView, Text, View } from "react-native";
+import React, { useState } from "react";
+import {
+  Linking,
+  Platform,
+  Pressable,
+  SafeAreaView,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+} from "react-native";
 import { useNavigation } from "@react-navigation/native";
+import CheckBox from "@react-native-community/checkbox";
+import { ToolbarActionButton } from "./ToolbarActionButton";
 
 const AddShippingZone = () => {
   const navigation = useNavigation();
 
+  const [isLimitEnabled, setLimitEnabled] = useState(false);
+
+  React.useEffect(() => {
+    navigation.setOptions({
+      headerRight: () =>
+        ToolbarActionButton({
+          label: "Save",
+          onPress: () => console.log("Toolbar action button pressed"),
+        }),
+    });
+  }, [navigation]);
+
+  const _renderPostCodes = function () {
+    if (isLimitEnabled) {
+      return (
+        <View style={{ marginTop: 10 }}>
```

Comment:
Do you think all of these margins should go in the styles object? Seems easy to modify them that way!

Reply:
Now when I think about this, I think that extracting things to style objects is continent when style is repeated. If it's a single case, I'd omit that to not have too many styles named "somethingContainer". WDYT?

Reply:
In general, I think it is easier to have a single place to edit styles and layout numbers. It makes it easy to know where to change the code. 

But it's up to you! 👍